hotline/notifications/templatetags/notifications.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+from hotline.notifications.models import UserNotificationQuery
+
+register = template.Library()
+
+
+@register.filter
+def notification_url(entry):
```

Comment:
I think because this isn't a model form, and the queryset isn't defined until the init method, there isn't a way to know that that particular field corresponds with a particular model. At least, that's what it seems like. The docs, and django's github don't reveal an easy way to access a model instance's attributes from within an iteration of a form's field, and most of the answers I've found so far suggest to use a template tag of some sort.